src/core/api.js

```diff
@@ -0,0 +1,40 @@
+import axios from "axios";
+import { API } from "./constants";
+
+const ApiService = {
+  init() {
+    axios.defaults.baseURL = API.BASE_URI;
+  },
+
+  getMoviesWithLimit(limit) {
+    const params = {
+      [API.KEYS.LIMIT]: limit
+    };
+
+    return axios
+      .get(API.MOVIES, { params })
+      .then(res => res.data)
+      .then(info => info.data);
+  },
+
+  getMoviesByParameters(sortBy, sortOrder, search, searchBy, offset, limit) {
```

Comment:
When you have so many parameters, it could be more convenient to pass them as an object - that way you won't have to think about their ordering.